cmd/root.go

```diff
@@ -44,8 +45,6 @@ const (
 `
 )
 
-var version = "dev"
```

Comment:
You still should use the version variable populated by ldflags when using a binary built by go build.